Day3/README.md

```diff
@@ -354,7 +354,37 @@ k=[i for i,j in zip(c,d) if i!=j]
 print("list of unmatched characters are:",k)
 print("number of characters not matching are:",len(k))
 
-```
+``` 
+/* @date 4/09/2020
+ * @author Shashwat Gupta (shashwatxdevelop)
+ */
+import java.util.Scanner;
+public class HammingDistance {
+
+	public static void main(String[] args) {
+		String s1, s2; int l1, l2, i, k=0; char ch = 0;
+		Scanner sc= new Scanner (System.in);
+		s1 = sc.nextLine();  //inputing string 1
+		s2 = sc.nextLine();  // imputing string 2
+		l1 = s1.length();    // finding length of string 1
+		l2 = s2.length();    // finding length of string 2
+		
+		if(l1==l2)        // comparing both lengths of strings because hamming distance works only on same length of string
+		{
+			for (i=0 ; i<l1; i++)   // for loop for extraction of characters from both of the strings
+			{ 
+				if (s1.charAt(i)!= s2.charAt(i))  // checking same character of both strings
+					k+ = 1;                       // if same character is not found change in variable k by 1
+				
+			}
+		}
+		System.out.println("Hamming Distance is " + k);   // printing the hamming distance
```

Comment:
Prints out that Hamming Distance is 1 when the lengths of the strings are unequal in place of displaying an error message. There should be an else clause to handle that case. Sorry I missed that out in my previous review. Do test your code with sample inputs (and perhaps your own test cases) to make sure it matches with the accepted solution.